docs/components/pages/repo-home/RepoHero.tsx

```diff
@@ -84,11 +84,11 @@ export function RepoHero() {
         </FadeIn>
         <FadeIn
           delay={0.3}
-          className="z-50 flex flex-col items-center w-full max-w-md gap-5 px-6"
+          className="z-50 flex flex-col items-center w-full max-w-md gap-5 px-6   mt-5 "
         >
-          <div className="flex flex-col w-full gap-3 md:!flex-row">
+          <div className="flex flex-col w-full gap-3 md:!flex-row  ">
             <CTAButton>
-              <Link href="/repo/docs" className="block py-3">
+              <Link href="/repo/docs" className="block py-3 text-base ">
```

Comment:
not a big deal, but can you remove the trailing white spaces from the className attributes?

Reply:
Thank you. I have removed all trailing spaces.